### DISCUSSION TITLE
Deselect radio choice on new click event?

### DISCUSSION THREAD

**Discussion by ApolloDevs** [alpinejs/alpine#1100] [1. Help]
For a form of mine, I'm attempting to filter what is shown depending on what is selected in a drop-down, with the radio buttons unselecting should the selected dropdown choice change. (to avoid the radio selecting a (now) hidden choice)

I thought I could use the AlpineJS tabs as a base for this, as it's conditionally hiding parts of the page, however changing the dropdown value appears to not destroy the currently selected option. 

I have made a codepen to demonstrate what I'm aiming for - https://codepen.io/miladevs/pen/dyOWNZP

I feel like I'm half way there, it's just that I'm not very good when it comes to JS. 

Thank you. :)

**Comment by KevinBatdorf** [comment]
Your `select` should look like this:

```html
<select x-model="category">
  <option value="1">Category 1</option>
  <option value="2">Category 2</option>
</select>
```
